cypress/projects/typescript/playroom.config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  components: './components.ts',
+  snippets: './snippets.ts',
+  outputPath: './dist',
+  openBrowser: false,
+  storageKey: 'playroom-example-typescript',
+  webpackConfig: () => ({
+    module: {
+      rules: [
+        {
+          test: /\.tsx?$/,
+          exclude: /node_modules/,
+          use: [
+            {
+              loader: require.resolve('babel-loader'),
+              options: {
+                presets: [
+                  require.resolve('@babel/preset-env'),
+                  [
+                    require.resolve('@babel/preset-react'),
+                    { runtime: 'automatic' },
+                  ],
```

Comment:
Do we need `preset-typescript` too?

Reply:
It seems to be working without it. But I added it anyway.

Reply:
It works without it because the main Webpack config handles it https://github.com/seek-oss/playroom/blob/720d542ab73d64573efccea8df77c5a8d7c8be0e/lib/makeWebpackConfig.js#L82-L83

Reply:
Ah

Reply:
It was not that, it was the root `.babelrc` that was making things work. Confirmed it doesn't work without `@babel/preset-react` in the [latest commit](https://github.com/seek-oss/playroom/pull/288/commits/ec0b0d7574a4e8758782f4d9ea1784a0a701012f).